app/forms/did_challenge_hmrc_form.rb

```diff
@@ -0,0 +1,3 @@
+class DidChallengeHmrcForm < BaseForm
```

Comment:
That makes sense.  I'm neutral otherwise, it just sounded a bit stilted when I read it for the first time.